nucypher/cli/characters/stake.py

```diff
@@ -147,7 +151,23 @@ def stake(click_config,
     # Make Stakeholder
     #
 
-    STAKEHOLDER = stakeholder_config.produce(initial_address=staking_address)
+    # First let's check whether we're dealing here with a regular staker or a preallocation staker
+    is_preallocation_staker = escrow or beneficiary_address
+
+    if staking_address and is_preallocation_staker:
```

Comment:
Should we also do an XOR check for `escrow` and `beneficiary_address`? i.e. it wouldn't make sense to have both defined, would it?

Reply:
I think it doesn't hurt, no need to raise an error for that.